android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/login/LoginScreen.kt

```diff
@@ -1,28 +1,113 @@
 package vn.luongvo.kmm.survey.android.ui.screens.login
 
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.material.*
-import androidx.compose.runtime.Composable
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.*
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
 import org.koin.androidx.compose.getViewModel
-import vn.luongvo.kmm.survey.Greeting
+import vn.luongvo.kmm.survey.android.R
+import vn.luongvo.kmm.survey.android.ui.common.*
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.typography
 import vn.luongvo.kmm.survey.android.ui.theme.ComposeTheme
+import vn.luongvo.kmm.survey.android.ui.theme.White50
 
 @Composable
 fun LoginScreen(viewModel: LoginViewModel = getViewModel()) {
-    Surface(
-        modifier = Modifier.fillMaxSize(),
-        color = MaterialTheme.colors.background
-    ) {
-        Text(text = Greeting().greeting())
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
+    LoginScreenContent(
+        email = email,
+        password = password,
+        onEmailChange = {
+            email = it
+            // TODO https://github.com/luongvo/kmm-survey/issues/8
+        },
+        onPasswordChange = {
+            password = it
+            // TODO https://github.com/luongvo/kmm-survey/issues/8
+        },
+    )
+}
+
+@Composable
+private fun LoginScreenContent(
+    email: String,
+    password: String,
+    onEmailChange: (String) -> Unit,
+    onPasswordChange: (String) -> Unit,
+) {
+    Box {
+        Box(modifier = Modifier.fillMaxSize()) {
+            DimmedImageBackground(imageRes = R.drawable.bg_login)
+        }
+        Image(
+            painter = painterResource(id = R.drawable.ic_nimble_logo),
+            contentDescription = null,
+            modifier = Modifier
+                .align(Alignment.Center)
+                .wrapContentHeight()
+                .offset(0f.dp, (-229f).dp)
```

Comment:
@luongvo Just curious, why don't we just use `.offset(0.dp, (-299).dp)`?

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/48/commits/2bf5858983ff990d311a04c1e83df985919b6207